tests/Unit/V1/FiltersBuilderTest.php

```diff
@@ -78,7 +78,7 @@ public function testBuildWithAllFields()
 
     public function testAddInvalidIncludedCategory()
     {
-        $this->expectException(\InvalidArgumentException::class);
+        $this->setExpectedException(\InvalidArgumentException::class);
```

Comment:
`setExpectedException` has been deprecated, it looks like this should just be `expectException` as per: https://phpunit.readthedocs.io/en/7.3/writing-tests-for-phpunit.html#testing-exceptions 

Can you clarify why this should be `setExpectedException`?

Reply:
composer.json requires phpunit/phpunit 4.8 as a minimum (likely to satisfy the php 5.5 support requirement).

[setExpectedException()](https://github.com/sebastianbergmann/phpunit/blob/18e5f52e8412d23e739f7d8744e177039860e800/src/Framework/TestCase.php#L462) is the appropriate method to use across phpunit `^4.8|^5.0`, as [expectException() wasn't introduced until 5.2](https://github.com/sebastianbergmann/phpunit/blob/09c74ffc299a6bf5e6d1444cdc44b4f98f5826da/src/Framework/TestCase.php#L507)

If the minimum php version is raised to 5.6, we could change the phpunit constraint to `^5.2` & restore the `expectException` method.